static/js/browser2/out_arc_section.tsx

```diff
@@ -174,4 +179,43 @@ export class OutArcSection extends React.Component<
     }
     return a > b ? 1 : -1;
   };
+
+  private fetchDataFromTriples(): void {
+    axios.get("/api/browser/triples/" + this.props.dcid).then((resp) => {
+      const triplesData = resp.data;
+      const outArcs = triplesData.filter(
+        (t) => t["subjectId"] == this.props.dcid
+      );
+      const outArcsByPredicateAndProvenance: OutArcData = {};
+      outArcs.forEach((outArc) => {
+        const predicate = outArc.predicate;
+        if (IGNORED_OUT_ARC_PROPERTIES.has(predicate)) {
+          return;
+        }
+        if (!outArcsByPredicateAndProvenance[predicate]) {
+          outArcsByPredicateAndProvenance[predicate] = {};
+        }
+        const provId = outArc.provenanceId;
+        if (!(provId in outArcsByPredicateAndProvenance[predicate])) {
```

Comment:
before this line, consider pulling out "outArcsByPredicateAndProvenance[predicate]" to a local var, which you can reference more easily here and below

Reply:
done

Reply:
done